lib/linux_admin/network_interface.rb

```diff
@@ -112,7 +114,11 @@ def netmask6(scope = :global)
     #
     # @return [String] IPv4 gateway address
     def gateway
-      @network_conf[:gateway]
+      @network_conf[:gateway4]
+    end
+
```

Comment:
Added documentation.

Previously, I was kinda reluctant to write it as the other documentation strings are not 100% accurate. (for instance, some stuff here relies on the appliance having single network_interface).